sovryn-token-bridge/bridge/migrations/12_deploy_allowToken.js

```diff
@@ -0,0 +1,10 @@
+const MultiSigWallet = artifacts.require("MultiSigWallet");
```

Comment:
I don't know how the migrations are used, but is it right to edit the old migration instead of creating a new one?

Reply:
cleanup of migrations folder and contracts folder